Source/UI/Validator/Validator.swift

```diff
@@ -1,5 +1,5 @@
 import Foundation
 
 protocol Validator {
-    func validate(text: String?) -> Bool
+    func isInvalid(value: Any?) -> Bool
```

Comment:
Not sure this is an improvement.
- we should avoid negation (is invalid vs is valid)
- why use Any? The validators are probably looking at Strings, why introduce ambiguity?

Reply:
I don't agree with Any. We should avoid ambiguity as much as we can, and there's not much more ambiguity then `Any`.

`isInvalid` is not clear because it negates a boolean. 

is invalid = is NOT valid
```swift
if isInvalid 

vs 

if isValid
```
easier to mess up logical boolean comparisons when the variable is already inverted.